src/domain/collaboration/innovation-flow-states/innovation.flow.state.service.ts

```diff
@@ -62,5 +62,14 @@ export class InnovationFlowStatesService {
         LogContext.INNOVATION_FLOW
       );
     }
+    // Avoid any characters JSON related:
+    // This validation is also performed on the client: domain/collaboration/InnovationFlow/InnovationFlowDragNDropEditor/InnovationFlowStateForm.tsx
+    // Keep them in sync consistently
+    if (stateNames.some(name => !/^[^[\],'"{}\\]+$/.test(name))) {
```

Comment:
- extract the pattern to a variable
- the tags should be a `simple-array` 
- because of them being strings, I believe we can enable most of the characters you have included